Mathlib/Data/Finsupp/Multiset.lean

```diff
@@ -110,8 +110,8 @@ theorem count_toMultiset [DecidableEq α] (f : α →₀ ℕ) (a : α) : (toMult
     _ = f.sum fun x n => n * ({x} : Multiset α).count a := by simp only [Multiset.count_nsmul]
     _ = f a * ({a} : Multiset α).count a :=
       sum_eq_single _
-        (fun a' _ H => by simp only [Multiset.count_singleton, if_false, H.symm, mul_zero]) fun H =>
-        by simp only [not_mem_support_iff.1 H, zero_mul]
+        (fun a' _ H => by simp only [Multiset.count_singleton, if_false, H.symm, mul_zero])
+        (by simp)
```

Comment:
Can you leave this simp squeezed? Is it now just `zero_mul`?

Reply:
As it happens, no, even though `zero_mul` does solve it:
```lean
        (by simp only [Multiset.mem_singleton, not_true, Multiset.nodup_singleton, Multiset.count_eq_one_of_mem,
          mul_one, implies_true])
```